src/core/src/CoreMain.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, argv):
                 if patch_assessment_successful and patch_installation_successful:
                     patch_installer.mark_installation_completed()
                     overall_patch_installation_operation_successful = True
+                elif patch_installer.set_patch_installation_status_to_warning_from_failed():
```

Comment:
A previous comment on this is still open: https://github.com/Azure/LinuxPatchExtension/pull/291#discussion_r2028921715
<img width="486" alt="image" src="https://github.com/user-attachments/assets/5a9dae1f-5adf-4d8e-83c0-067ee3c3d33a" />